sources/lib/Model/FlexibleEntity/ModifiedColumnEntityTrait.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/*
+ * This file is part of the PommProject/ModelManager package.
+ *
+ * (c) 2014 - 2016 Grégoire HUBERT <hubert.greg@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PommProject\ModelManager\Model\FlexibleEntity;
+
+/**
+ * ModifiedColumnEntityTrait
+ *
+ * @package     ModelManager
+ * @copyright   2014 - 2015 Grégoire HUBERT
+ * @author      Mikael Paris
+ * @license     X11 {@link http://opensource.org/licenses/mit-license.php}
+ * @see         FlexibleEntityInterface
+ */
+trait ModifiedColumnEntityTrait
+{
+    private $modified_columns = [];
+
+    /**
+     * @return array
+     */
+    public function getModifiedColumns()
+    {
+        return $this->modified_columns;
+    }
+
+    /**
+     * AddModifiedColumn
```

Comment:
Only return type is essential. Other comments is a duplicated (= bad) information.